File: OOPS/Abstract.java
abstract class  Parent 
{
    abstract void greet();
}
class child extends Parent
{
    void greet()
    {
        System.out.println("Welcome to Datasirpi");
    }
}
class Abstract
{
    public static void main(String args[])
    {
        child ch =new child();
        ch.greet();
    }
}

File: OOPS/Abstract.php
<?php
abstract class Sample 
{
	abstract function Greeting();
	abstract function PrintName();
}
class Main extends Sample
{
	public function Greeting()
	{
		 echo "Welcome to Datasirpi";
	}
	public function PrintName()
	{
		echo "Manikandan";
	}
}
$obj=new Main();
$obj->Greeting();
echo "<br>";
$obj->PrintName();
?>